drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/utils/HttpClient.java

```diff
@@ -22,6 +27,7 @@ public Response get(String url, Map<String, String> headers) throws IOException
         try (CloseableHttpClient client = createHttpClient()) {
             final HttpGet request = new HttpGet(url);
             addHeaders(headers, request);
+            log.info("Sending request: " + request + " with headers: " + Arrays.asList(request.getAllHeaders()));
```

Comment:
I'm not sure about this. WDYT? which level would be more appropriate? Debug? Fine?